oncotext/utils/dataset_factory.py

```diff
@@ -11,30 +13,58 @@ def get_oncotext_dataset_train(all_reports, label_maps, args, text_key, vocab_si
     train_reports = reports[: split_indx]
     dev_reports = reports[split_indx:]
 
-    train_data = oncotext.datasets.patholgy_dataset.PathologyDataset(
-                                                args,
-                                                train_reports,
-                                                label_maps,
-                                                text_key,
-                                                'train',
-                                                vocab_size=vocab_size)
-    dev_data = oncotext.datasets.patholgy_dataset.PathologyDataset(
-                                                args,
-                                                dev_reports,
-                                                label_maps,
-                                                text_key,
-                                                'dev',
-                                                vocab_size=vocab_size)
+    if label_maps[args.aspect][0] == "NUM":
+        dataset_obj = oncotext.datasets.pathology_tagging_dataset.PathologyTaggingDataset
+    else:
+        dataset_obj = oncotext.datasets.pathology_classification_dataset.PathologyClassificationDataset
+        
+    train_data = dataset_obj(args, train_reports, label_maps, text_key, 'train')
+    dev_data = dataset_obj(args, dev_reports, label_maps, text_key, 'dev')
+    
     return train_data, dev_data
 
 
-
-
-def get_oncotext_dataset_test(reports, label_maps, args, text_key, vocab_size):
-    test_data = oncotext.datasets.patholgy_dataset.PathologyDataset(args,reports, label_maps, text_key, 'test', vocab_size=vocab_size)
+def get_oncotext_dataset_test(reports, label_maps, args, text_key):
+    if label_maps[args.aspect][0] == "NUM":
+        dataset_obj = oncotext.datasets.pathology_tagging_dataset.PathologyTaggingDataset
+    else:
+        dataset_obj = oncotext.datasets.pathology_classification_dataset.PathologyClassificationDataset
+        
+    test_data = dataset_obj(args, reports, label_maps, text_key, 'test')        
     return test_data
 
+
 def get_embedding_tensor(config, args):
     embeddings =  pickle.load(open(config['EMBEDDING_PATH'],'rb'))
     args.embedding_dim = embeddings.shape[1]
     return embeddings
+
+
+def predsToLabels(preds, test_data, label_maps, diagnosis, args, text_key, logger):
```

Comment:
nitpicky but 

all our function names are "blah_blah", so please change this to that convention instead of Camel case.